docs/setup-dev-environment.md

```diff
@@ -48,138 +121,285 @@ rm node-$NODE_VERSION-$DISTRO.tar.xz
 
-`sudo locale-gen en_US.UTF-8`
+```bash
+sudo locale-gen en_US.UTF-8
+```
 
 Configure the database to listen on network connection.
 
-`sudo sed -i "s/#\?listen_address.*/listen_addresses '*'/" /etc/postgresql/10/main/postgresql.conf`
+```bash
+sudo sed -i "s/#\?listen_address.*/listen_addresses '*'/" /etc/postgresql/12/main/postgresql.conf
+```
 
 Allow authenticated connections from any IP (so includes the host).
 
-`echo "host    all             all             all                     md5" | sudo tee --append /etc/postgresql/10/main/pg_hba.conf > /dev/null`
+```bash
+echo "host    all             all             all                     md5" | sudo tee --append /etc/postgresql/12/main/pg_hba.conf > /dev/null
+```
 
 Restart postgres to pick up config changes.
 
-`service postgresql restart`
+```bash
+service postgresql restart
+```
 
 Create a superuser role for this user (`<username>`) if it does not already exist. The
 password `<pgpassword>` is arbitrary and probably should not be your Ubuntu login password.
 
-`sudo -u postgres psql -c "SELECT 1 FROM pg_user WHERE usename = '<username>';" | grep -q 1 || sudo -u postgres psql -c "CREATE ROLE <username> SUPERUSER LOGIN PASSWORD '<pgpassword>';"`
+```bash
+sudo -u postgres psql -c "SELECT 1 FROM pg_user WHERE usename = '<username>';" | grep -q 1 || sudo -u postgres psql -c "CREATE ROLE <username> SUPERUSER LOGIN PASSWORD '<pgpassword>';"
+```
+
+<details>
+<summary>Note for "Colouring London" devs</summary><p></p>
 
-Create a colouring london database if none exists. The name (`<colouringlondondb>`) is arbitrary.
+If you intend to load the full CL database from a dump file into your dev environment, run the above `psql` command with `<username>` as "cldbadmin" and use that username in subsequent steps, but also run the above a second time with `<username>` as "clwebapp" (see section [:house: Loading the building data](#house-loading-the-building-data) for more details).
 
+</details><p></p>
+
+Set environment variables, which will simplify running subsequent `psql` commands.
+
+```bash
+export PGPASSWORD=<pgpassword>
+export PGUSER=<username>
+export PGHOST=localhost
+export PGDATABASE=<colouringlondondb>
 ```
+
+Create a colouring london database if none exists. The name (`<colouringlondondb>`) is arbitrary.
+
+```bash
 sudo -u postgres psql -c "SELECT 1 FROM pg_database WHERE datname = '<colouringlondondb>';" | grep -q 1 || sudo -u postgres createdb -E UTF8 -T template0 --locale=en_US.utf8 -O <username> <colouringlondondb>
 ```
 
-To test the app user's connection to the database, you could run `psql` interactively:
+Run `psql` interactively.
 
+```bash
+psql
 ```
-psql -d <colouringlondondb> -U <username> -h localhost
+
+In `psql`, necessary postgres extensions.
+
+```bash
+create extension postgis;
+create extension pgcrypto;
+create extension pg_trgm;
 ```
 
-Create the necessary postgres extensions.
+Then quit `psql` by typing `\q` and hitting return.
```

Comment:
what about replacing it with

```
psql -d <colouringlondondb> -c "create extension postgis;"
psql -d <colouringlondondb> -c "create extension pgcrypto;"
psql -d <colouringlondondb> -c "create extension pg_trgm;"
```

?

Reply:
I think this makes sense, I had a reason why I changed it, however I think that reason is no longer relevant

Reply:
I can confirm that it works fine.